devtools-integration-tests/testerina-integration-tests/.gitignore

```diff
@@ -0,0 +1 @@
+target
```

Comment:
Add end of line

Reply:
Removed gitignore file since we wont be building the project with `build`. Most cases we will use `test` or `build -c`